teachopencadd/talktorials/001_query_chembl/talktorial.ipynb

```diff
@@ -171,31 +201,41 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
     "import math\n",
-    "import os.path\n",
+    "from pathlib import Path\n",
     "from zipfile import ZipFile\n",
     "from tempfile import TemporaryDirectory\n",
     "\n",
```

Comment:
Dominique suggested that no need to add new lines between the third-party imports, so I just follow her.